test/runtests.jl

```diff
@@ -854,7 +854,7 @@ end
     @test QuadGK.quadgk(sin, -y, y)[1] ≈ cos(-y) - cos(y) atol = 2 * eps(Float64)
     @test QuadGK.quadgk(exp, 0.4, x)[1] ≈ exp(x) - exp(0.4)
     @test QuadGK.quadgk(sin, w, 2.7)[1] ≈ cos(w) - cos(2.7)
-    @test QuadGK.quadgk(t -> cos(x - t), 0, 2pi)[1] ≈ measurement(0) atol = 2e-13
+    @test QuadGK.quadgk(t -> cos(x - t), 0, 2pi; atol=1e-16)[1] ≈ measurement(0) atol = 5e-16
```

Comment:
Asking for better than machine precision may be a bit dicey.

Reply:
Oh, I followed the example in the first version of https://github.com/JuliaPhysics/Measurements.jl/issues/134#issuecomment-1432280499 (which is what I saw in the email notification) :slightly_smiling_face:  Fixed